File: iosApp/iosApp/DesignSystem/Dimen.swift
import CoreGraphics

/// 간격 — 4pt 베이스. 안드로이드 `TeamFisSpacing` 과 값이 같아야 한다.
enum TeamFisSpacing {
    static let xs: CGFloat = 4
    static let sm: CGFloat = 8
    static let md: CGFloat = 12
    static let lg: CGFloat = 16
    static let xl: CGFloat = 20
    static let xxl: CGFloat = 24
    static let xxxl: CGFloat = 32

    /// 화면 좌우 여백
    static let screenHorizontal: CGFloat = 20
}

enum TeamFisRadius {
    static let card: CGFloat = 4
}

enum TeamFisSize {
    static let header: CGFloat = 56
    static let minTouchTarget: CGFloat = 44

    /// 알약 칩 높이 — 펼쳐보기 같은 것
    static let chip: CGFloat = 36

    /// 목록 한 줄의 최소 높이 — 글이 짧아도 줄이 납작해지지 않게
    static let listRowMin: CGFloat = 56

    /// 스크롤 맨 아래에 두는 여백 — **접힌 하단 유리 바가 마지막 줄을 덮지 않을 만큼**.
    /// 시스템이 잡아 주는 여백은 펴짐 기준이라, 접히며 줄어드는 만큼(~60)을 더해 둔다.
    ///
    /// **줄이지 말 것** — 줄이면 끝까지 내렸을 때 마지막 줄이 바에 가린다 (2026-09-07).
    /// 안드로이드에는 없는 값이다 (거기 탭 바는 콘텐츠 위에 안 뜬다).
    static let bottomBarClearance: CGFloat = 88
}

File: androidApp/src/main/kotlin/com/teamfis/app/ui/theme/Dimen.kt
package com.teamfis.app.ui.theme

import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.ui.unit.dp

/** 간격 — 4pt 베이스 */
object TeamFisSpacing {
    val xs = 4.dp
    val sm = 8.dp
    val md = 12.dp
    val lg = 16.dp
    val xl = 20.dp
    val xxl = 24.dp
    val xxxl = 32.dp

    /** 화면 좌우 여백 */
    val screenHorizontal = 20.dp
}

object TeamFisRadius {
    val card = RoundedCornerShape(4.dp)
    val full = RoundedCornerShape(percent = 50)
}

object TeamFisSize {
    val header = 56.dp
    val minTouchTarget = 44.dp

    /** 알약 칩 높이 — 펼쳐보기 같은 것 */
    val chip = 36.dp

    /** 목록 한 줄의 최소 높이 — 글이 짧아도 줄이 납작해지지 않게 */
    val listRowMin = 56.dp

    /** 세그먼트 바 높이 (신규/재등록 같은 모드 고르개) */
    val segment = 48.dp

    /** 화면 아래 고정 버튼 */
    val actionButton = 52.dp
}
